### DISCUSSION TITLE
Tuya Integration with unsupported devices (Soil Sensor, Humidifier)

### DISCUSSION THREAD

**Discussion by rsmatias** [tuya/tuya-home-assistant#896] [Q&A]
Hi all,
I've 2 sensors connected to Home Assistant trough Tuya Integration. Both are stated as UNSUPPORTED.
After I had a quick look at the Tuya IoT Cloud website and realized that the Product Category are:

   - Tuya ZigBee Soil Sensor: "wsdcg" 
   - Pro Breeze 30L humidifier: "cs"

None of these two Product Categories are present in the Tuya Integration file named sensor.py. I believe that this is why Tuya Integration does not identify them.
My question is: Who is the right contributor the can help add these Product Categories to the file and help testing these devices?

Thanks a lot in advance.
Regards,
Rodrigo Matias

**Comment by figarski** [comment]
I'd like to join this one adding one item: Water Valve: "ggq"
